src/controllers/user.controller.ts

```diff
@@ -139,4 +139,58 @@ export async function setUserCuil(req: Request, res: Response) {
   } catch (error) {
     handleHttpError(res, 'cannot set user cuil')
   }
-}
\ No newline at end of file
+}
+
+export async function getBenefitNumber(req: Request, res: Response) {
+  try {
+    const { from: number, body: message }: Ctx = req.body.ctx;
+
+    const user = await models.user.findOne({cellphone: number});
+    
+    if (!user) {
+      return handleHttpError(res, 'user not found');
+    };
+
+    if(!user.CUIT) {
+      return handleHttpError(res, 'user cuit not found')
+    }
+    
+    user.benefitNumber = message;
+    await user.save();
+
+    res.status(200).send({message: 'benefit number saved'})
```

Comment:
same here, remember how is used the messages, see the other controllers in user.ts